shared/fs/footer/container.js

```diff
@@ -16,7 +16,11 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({
 
 const mergeProps = (stateProps, {opener, dismisser}, ownProps) =>
   ({
-    downloads: Array.from(stateProps.transfers.filter(transferState => transferState.type === 'download'))
+    downloads: Array.from(
```

Comment:
don't think Array.from is required here

Reply:
Hmm `stateProps.transfers` is `Map`, and we'd need an `Array` to sort right?

Reply:
Array.from is fine per our discussion